inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -61,6 +59,10 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*decomposition1_batch=5_hidden_size=10_input_size=30_.*tanh.relu.*_clip=0_linear_before_reset=1.*_targetDevice=CPU_.*)",
         // TODO: Issue 47556. [NGraph] CTCGreedyDecoderSeqLen. Invalid type transformation i64 to i32.
         R"(.*(CTCGreedyDecoderSeqLenLayerTest).*(idxPRC=I64).*)",
+        // TODO: Issue: 48096
```

Comment:
The problem with changing original constant names was resolved in https://github.com/openvinotoolkit/openvino/pull/4143/files#diff-729914533c44221a8064a8afcbe95002092488741b5459f3d49ef4ba62c73f85